docs/commands.md

```diff
@@ -521,13 +522,24 @@ JSON.ARRTRIM <key> <path> <start> <stop>
 
 #### Description
 
-Trim an array so that it contains only the specified inclusive range of elements.
+Trims an array so that it contains only the specified inclusive range of elements.
+
+This command is extremely forgiving and using it with out-of-range indexes will not produce an error. There are a few differences between how RedisJSON v2.0 and legacy versions handle out-of-range indexes.
+
+Behavior as of RedisJSON v2.0:
```

Comment:
don't we have versioned documentation?  SHouldn't a note be sufficient that this changed compared to v1?

Reply:
I see what you mean. I will remove the "Legacy behavior" part of my changes.